htscodecs_bundled.mk

```diff
@@ -25,6 +25,7 @@
 
 HTSCODECS_SOURCES = $(HTSPREFIX)htscodecs/htscodecs/arith_dynamic.c \
         $(HTSPREFIX)htscodecs/htscodecs/fqzcomp_qual.c \
+	$(HTSPREFIX)htscodecs/htscodecs/htscodecs.c \
```

Comment:
This is the only line using a tab in this file.  All others are using spaces.

Reply:
Oops, tab has now gone.